src/Geno/C++/GUI/Modals/ProjectSettingsModal.cpp

```diff
@@ -72,7 +72,7 @@ void ProjectSettingsModal::UpdateDerived( void )
 		return;
 	}
 
-	Project* pProject = pWorkspace->ProjectByName( m_EditedProject );
+	Project* pProject = ( Project* )pWorkspace->ChildByName( m_EditedProject );
```

Comment:
Bold to assume that it's a project we got in return! We could have a file filter with the same name as one of the projects. Maybe make it so that ChildByName can take an optional NodeType parameter and only look for that node type?